tests/page/page-screenshot.spec.ts

```diff
@@ -615,3 +615,45 @@ it.describe('page screenshot animations', () => {
     ]);
   });
 });
+
+it.only('should work', async ({ page, server }) => {
```

Comment:
```suggestion
it.only('should work', async ({ page, server }) => {
```
Quick tip for development:

npm run ctest foo.spec.ts:42

Instead of test.only

Reply:
Sometimes I do fallback to line numbers, but do find .only more convenient in the end of the day